frontend/src/ToDoGallery.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {useEffect, useState} from 'react';
+import {status, ToDo} from "./models/model";
+import ToDoItem from "./ToDoItem";
+
+
+export default function ToDoGallery() {
+
+    const [toDoItems, setToDoItems] = useState([] as Array<ToDo>)
+    const [id, setId] = useState('')
+    const [task, setTask] = useState('')
+    const [description, setDescription] = useState('')
+    const [stati, setStati] = useState(status.Open)
+
+    const requestBody = {
```

Comment:
Das würde ich in die`setAttributes`-Funktion kopieren, damit das Objekt erst dann erzeugt wird, wenn es tatsächlich benötigt wird.